ocs_ci/ocs/cluster.py

```diff
@@ -833,6 +833,37 @@ def validate_ocs_pods_on_pvc(pods, pvc_names, pvc_label=None):
             ), f"No PVC {pvc_name} found for pod: {pod_name} in PVCs: {pvc_names}!"
 
 
+@retry(CommandFailed, tries=3, delay=10, backoff=1)
+def validate_claim_name_match_pvc(pvc_names, validated_pods=None):
+    """
+    Validate if OCS pods have mathching PVC and Claim name
+
+    Args:
+        pvc_names (list): names of all PVCs you would like to validate with.
+        validated_pods(set): set to store already validated pods - if you pass
+            empty set outside of this function, this will speed up the next
+            validation when re-tries as it will skip those already validated
+            pods stored in this set.
+    Raises:
+        AssertionError: when the claim name does not match one of PVC name.
+
+    """
+    if validated_pods is None:
+        validated_pods = set()
```

Comment:
```suggestion
    validated_pods = validated_pods or set()
```

Reply:
This will not work. As I would like to pass empty set outside of this function.
As it's a mutable object, it will still refer to the same one and each iteration it will adding pods to the same object.

Empty set will be validated as False so in this  case you will get always empty set from inside the function which will get each iteration re-created. 

see:
```
In [1]: set() or "You don't want this"
Out[1]: "You don't want this"
``` 
Hence I really need to validate if it's None.

Reply:
Got it, that makes sense! LGTM then